doc/docs/master/deploy-manage/deploy/loki.md

```diff
@@ -0,0 +1,68 @@
+!!! note To deploy and configure a Pachyderm cluster
+to forward logs to Loki, 
+a ***Pachyderm Enterprise License*** is required. 
+
+# Enabling Loki
+
+Enabling Loki logging is done by setting the `LOKI_LOGGING` [environment variable](https://docs.pachyderm.com/latest/deploy-manage/deploy/environment-variables/) to `true` on the pachd container.
+
+## Shipping logs to Loki
+
+Loki retrieves logs from pods in Kubernetes through
+an agent service called **Promtail**. 
+**Promtail** runs on each node and
+sends logs from Kubernetes pods to the Loki API Server,
+tagging each log entry with information
+about the pod that produced it. 
+
+You need to [configure Promtail](https://grafana.com/docs/loki/latest/clients/promtail/configuration/) for your environment
+to ship logs to your Loki instance. 
+If you are running **multiple nodes**, 
+then you will need to install and configure Promtail
+**for each node** shipping logs to Loki.
+
+
+## Fetching logs
+
+While enabling Loki will enable the collection of logs, commands such as `pachctl logs` will not fetch logs directly from Loki until the `LOKI_SERVICE_HOST` and `LOKI_SERVICE_PORT` environment variables are set on the `pachd` container.
+
+For example, a `deployment.json` generated with 
+```shell
+    pachctl deploy local --dry-run > deployment.json`     
+```
+can be modified to include log forwarding as follows:
```

Comment:
can be modified to include log forwarding as follows: -> can be modified to make logs available for Loki as follows: